### ISSUE TITLE
Support multiple policies for a resource

### ISSUE THREAD

**Issue by i4rilu** [NWebsec/NWebsec#128]
https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Content-Security-Policy#Multiple_content_security_policies

**Comment by klings** [comment]
You can already set both the CSP and CSP-Report-Only header with NWebsec. Can you explain the scenario here?

**Comment by i4rilu** [comment]
Scenario is related with script-src '**strict-dynamic**' option.
https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Content-Security-Policy/script-src#strict-dynamic

When '**strict-dynamic**' option is enabled, "*any whitelist or source expressions such as 'self' or 'unsafe-inline' will be ignored*". This makes impossible to load any inline JavaScript (even signed with nonce/hash).

To solve this scenario, another CSP or CSP-Report-Only header is needed:
script-src 'unsafe-inline'

NWebsec does not support multiple CSP and/or CSP-Report-Only headers in a configuration / HTTP response so far.

**Comment by klings** [comment]
I don't think that would work. The standard specifies that content must be allowed by all policies: https://www.w3.org/TR/CSP3/#multiple-policies

In this case the content should be blocked by the first policy.

**Comment by Sora2455** [comment]
> In this case the content should be blocked by the first policy.

Not according [to Google](https://csp.withgoogle.com/docs/faq.html#strict-dynamic-with-whitelists):

> I want to use 'strict-dynamic', but also enforce a URL whitelist; is this possible?
> 
> On its own, 'strict-dynamic' causes the browser to rely only on nonces or hashes and ignore the URI whitelist in script-src, for backwards compatibility reasons. However, it's possible to enforce both a whitelist and nonces with 'strict-dynamic' by setting two policies:
> ```
> Content-Security-Policy: script-src foo.example.org bar.example.org 'unsafe-inline'
> Content-Security-Policy: script-src 'nonce-random123' 'strict-dynamic' 'unsafe-inline' https:
> ```
> The browser will check each script against each policy separately and only allow those which match both policies. In this case, it will accept scripts with a nonce of random123 (second policy) which are either inline <script> blocks or come from {foo,bar}.example.org (first policy).
